core/config/engine.cpp

```diff
@@ -74,6 +74,14 @@ int Engine::get_max_fps() const {
 	return _max_fps;
 }
 
+void Engine::set_audio_output_latency(int p_msec) {
```

Comment:
Why having setter & getter here? Is it because it's faster to query this than project settings?

Reply:
I needed a Engine setter and getter because the project setting was directly queried in the audio drivers. There was no way to override it without preventing the changed setting from being written to `project.godot` if you use `ProjectSettings.save()` (which happens if you run the editor with this CLI argument).